### ISSUE TITLE
Web interface not coming up with 1.21 (latest) image

### ISSUE THREAD

**Issue by eroji** [haugene/docker-transmission-openvpn#430]
Pulled the latest image today and web UI suddenly stopped working.

Docker logs show
```
transmission auth required
[2018-02-15 16:21:03.543] transmission-remote:  (http://localhost:9091/transmission/rpc/) Couldn't connect to server
Update UFW rules before changing port in Transmission
denying access to 
ERROR: Bad port
allowing 33222 through the firewall
Rule added
[2018-02-15 16:21:03.890] transmission-remote:  (http://localhost:9091/transmission/rpc/) Couldn't connect to server
Checking port...
[2018-02-15 16:21:13.910] transmission-remote:  (http://localhost:9091/transmission/rpc/) Couldn't connect to server
```

Within the container iptables show
```
Chain ufw-user-input (1 references)
target     prot opt source               destination         
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            multiport dports 49152:65535
ACCEPT     tcp  --  172.17.0.1           0.0.0.0/0            tcp dpt:9091
ACCEPT     tcp  --  172.17.0.0/16        0.0.0.0/0            tcp dpt:9091
ACCEPT     tcp  --  0.0.0.0/0            0.0.0.0/0            tcp dpt:33222
ACCEPT     udp  --  0.0.0.0/0            0.0.0.0/0            udp dpt:33222
```

**Reverting to 1.19 brought the UI back up.**

**Comment by haugene** [comment]
Yup, there has been some issues on this lately. Guessing it's basically the same as #426 and #428. You could also try the dev tag, going to promote that to the next release soon. Hopefully things are working well there, and that a new release will resolve this.

**Comment by mgrandl** [comment]
I had the same issue and switching to dev tag fixed it. Thanks!

**Comment by mbach04** [comment]
I'm not getting the transmission web UI with 1.21, 1.19, or dev tag while running on a synology box. However, running on my macbook I can reach the web ui without an issue using dev tag.

**Comment by haugene** [comment]
Dev is now merged to master and the latest tag should be ok. Closing this, reopen if the problem persists.

**Comment by haugene** [comment]
There were some troubles in the 1.20-range due to the RPC whitelist option that was introduced in the new version of transmission. But as far as I know this should be fixed in version 2.0 and up, so latest should also have worked...
